mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -2429,8 +2438,13 @@ LogicalResult ModuleImport::processFunction(llvm::Function *func) {
   SmallVector<llvm::BasicBlock *> reachableBasicBlocks;
   for (llvm::BasicBlock &basicBlock : *func) {
     // Skip unreachable blocks.
-    if (!reachable.contains(&basicBlock))
+    if (!reachable.contains(&basicBlock)) {
+      if (basicBlock.hasAddressTaken())
+        emitError(funcOp.getLoc())
```

Comment:
```suggestion
        return emitError(funcOp.getLoc())
```
We probably also want to return failure to ensure the translations stops?